djstripe/migrations/0005_auto_20190703_0958.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 2.2.2 on 2019-07-03 09:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+import djstripe.enums
+import djstripe.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('djstripe', '0004_auto_20190612_0850'),
+    ]
+
+    operations = [
+		# Altering the index is required because while we chnage the name of old PaymentMethod model to
+		# DjStripePaymentMethod, the migrations didn't update the names of index.
+		# In the current migration, we create a new PaymentMethod model, hence before creating it, its
+		# better to rename the old index.
+		migrations.RunSQL(
+			'ALTER INDEX djstripe_paymentmethod_id_0b9251df_like rename TO djstripe_paymentmethod_legacy_id_0b9251df_like',
+			'ALTER INDEX djstripe_paymentmethod_legacy_id_0b9251df_like rename TO djstripe_paymentmethod_id_0b9251df_like',
```

Comment:
Ah, I think this command is postgres specific, I'll see if I can change it to work on mysql (and hopefully sqlite too).

Reply:
done in https://github.com/dj-stripe/dj-stripe/pull/914/commits/d56dc5c05a84c25bc9f8f51c2baf7595a3fdce0e